lib/eunit/test/tutf8.erl

```diff
@@ -0,0 +1,12 @@
+%% coding: utf-8
+
+-module(tutf8).
+
+-include_lib("eunit/include/eunit.hrl").
+
+foo_test_() ->
+	[
+ 	  {<<"_göod">>, fun() -> io:format("goöd"), io:format([128,64]), ?assert("gö"=="gö") end}
```

Comment:
<<"_göod">>   should be <<"_göod/utf8>> if you want the binary to be utf-8 encoded (and if you do the test fails) 
which shows that something is fundamentally wrong here.